backend/typescript/models/role.model.ts

```diff
@@ -0,0 +1,7 @@
+import { Column, DataType, Model, Table } from "sequelize-typescript";
+
+@Table({ tableName: "roles" })
```

Comment:
Maybe remove timestamps? Although I guess it doesn't matter since you're never really modifying this table

Reply:
true, good catch